manifests/init.pp

```diff
@@ -404,6 +407,7 @@
   Optional[String[1]] $ip_version                                       = undef,
   Optional[Enum['yes', 'no']] $clear_node_high_bit                      = undef,
   Optional[Integer] $max_messages                                       = undef,
+  Optional[String] $test_corosync_config_cmd                            = undef,
```

Comment:
I still think this should be named `config_validate_cmd` with the default value of `'/usr/bin/env COROSYNC_MAIN_CONFIG_FILE=% /usr/sbin/corosync -t'`

Reply:
```suggestion
  String[1] $config_validate_cmd                                        = '/usr/bin/env COROSYNC_MAIN_CONFIG_FILE=% /usr/sbin/corosync -t',
```